### PR TITLE
Global exception handler

### PR DESCRIPTION
**Pull request by vrudkovsky**

### CODE REVIEW

File: src/main/java/com/vpcomtek/onlinebookstore/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
     private String title;
+    @NotBlank
+    @Size(max = 255, message = "Text limit 255 symbols")
     private String author;
+    @NotBlank
+    @Pattern(regexp = "^(?:[0-9-]{10}|[0-9-]{13})$",
+            message = "Text limit 10 or 13 symbols, without any letters!")
     private String isbn;
+    @NotBlank
+    @Min(0)
```
Comment (Elena-Bruyako) [inline src/main/java/com/vpcomtek/onlinebookstore/dto/CreateBookRequestDto.java]:
```suggestion
    @Positive
```

File: src/main/java/com/vpcomtek/onlinebookstore/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
+    @NotBlank
+    @Size(max = 255, message = "Text limit 255 symbols")
     private String title;
+    @NotBlank
+    @Size(max = 255, message = "Text limit 255 symbols")
     private String author;
+    @NotBlank
+    @Pattern(regexp = "^(?:[0-9-]{10}|[0-9-]{13})$",
```
Comment (Elena-Bruyako) [inline src/main/java/com/vpcomtek/onlinebookstore/dto/CreateBookRequestDto.java]: It's better to create custom annotation @Isbn and create Validator for it

File: src/main/java/com/vpcomtek/onlinebookstore/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -1,15 +1,28 @@
...
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
```
Comment (Elena-Bruyako) [inline src/main/java/com/vpcomtek/onlinebookstore/dto/CreateBookRequestDto.java:13]: You can use record for DTO

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! Let's improve your solution!

**Review by Rommelua — APPROVED** [review]

**Pull request merged**